package.json

```diff
@@ -611,6 +611,16 @@
           "default": true,
           "description": "Adds a space after a separator (',' and ';')."
         },
+        "powershell.codeFormatting.WhitespaceInsideBrace": {
+          "type": "boolean",
+          "default": true,
+          "description": "Adds a space after an opening brace ('}') and before a closing brace ('}')."
```

Comment:
You say "opening brace" but then use a closing brace as an example.  I think that should be `('{')`.

Reply:
Well spotted, thanks. I fixed it